images/ipi-deprovision/Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM registry.svc.ci.openshift.org/openshift/hive-v4.0:hive
 LABEL maintainer="hongkliu@redhat.com"
 
+ARG JQ_VERSION=1.6
+
 COPY cmd/ipi-deprovision/ipi-deprovision.sh  /usr/bin/ipi-deprovision.sh
 
 # Install awscli
 RUN chmod +x /usr/bin/ipi-deprovision.sh && \
       curl -O https://bootstrap.pypa.io/get-pip.py && \
       python get-pip.py && \
       rm -fv get-pip.py && \
-      pip install awscli==1.16.202
+      pip install awscli==1.16.202 && \
+      curl -L -o jq "https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64" && \
```

Comment:
On Fedora at least, `jq` is an RPM -- is it not on CentOS?

Reply:
Or from `pip`?

Reply:
only my 2 cents ... our base image is our beloved rhel image.
`jq` in fedora/centos can come with yum/dnf. In the case of centos, it comes from epel-release which i do not want to install :) https://serverfault.com/questions/768026/how-to-install-jq-on-rhel6-5

Did not go for with fedora although the jq in my pc is from dnf.

Reply:
Forgot to mention, i cannot test this locally with docker image directly because the yum repo url points to a svc in cluster. Silly me, I must have known this already.

Reply:
OK, that makes sense